manifests/package.pp

```diff
@@ -45,14 +45,20 @@
       $repo_require = undef
 
       $pkg_version = inline_template("<%= scope.lookupvar('sensu::version').sub(/(.*)\\-/, '\\1.') %>")
-      $pkg_title = 'Sensu'
-      $pkg_name = 'Sensu'
-      $pkg_source = "C:\\Windows\\Temp\\sensu-${sensu::version}.msi"
-      $pkg_require = "Remote_file[${pkg_source}]"
+      # Download the latest published version
+      $pkg_url_version = $::sensu::version ? {
+        installed => 'latest',
+        default   => $pkg_version,
+      }
+      $pkg_title = 'sensu'
+      $pkg_name = 'sensu'
```

Comment:
tl;dr $pkg_name should be Sensu on windows to match Add/Remove Programs, $pkg_title should be sensu to match internal catalog relationships.

$pkg_name and $pkg_title are both necessary because the title is for internal Puppet references and relationships.  This provides Package[sensu] consistently across platforms.  On windows, the name of the MSI in Add/Remove programs is Sensu (capitalized).  The MSI provider needs the name parameter to line up with Add/Remove programs, otherwise it will always report the package as absent.